_data/sites.json

```diff
@@ -5477,6 +5477,17 @@
         ]
     },
 
+    {
+        "name": "PaperKarma",
+        "url": "https://www.paperkarma.com/faq",
+        "difficulty": "hard",
+        "email": "human@paperkarma.com",
+        "notes": "Send an email to human@paperkarma.com requesting deletion. If you have a subscription plan through the App Store or Google Play, be sure to cancel your subscription in the proper store.",
+        "domains": [
+            "paperkarma.com"
+        ]
+    },
+
```

Comment:
This should be before `Parcello` to keep sorted

Reply:
Corrected.